packages/storybook/stories/VideoTile/VideoTile.stories.tsx

```diff
@@ -71,6 +72,16 @@ const getDocs: () => JSX.Element = () => {
         <VideoTileMenuItemsExample />
       </Canvas>
 
+      <Heading>Reaction Animation for Video Tile</Heading>
```

Comment:
This should be a separate PR as part of the DoD work.

Reply:
Will follow this feedback for the Send Reaction PR flow.